optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/java/org/optaplanner/workbench/screens/solver/backend/server/ToSolverConfigTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.optaplanner.workbench.screens.solver.backend.server;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.optaplanner.core.config.constructionheuristic.ConstructionHeuristicPhaseConfig;
+import org.optaplanner.core.config.constructionheuristic.ConstructionHeuristicType;
+import org.optaplanner.core.config.score.definition.ScoreDefinitionType;
+import org.optaplanner.core.config.solver.SolverConfig;
+import org.optaplanner.workbench.screens.solver.model.ConstructionHeuristicPhaseConfigModel;
+import org.optaplanner.workbench.screens.solver.model.ConstructionHeuristicTypeModel;
+import org.optaplanner.workbench.screens.solver.model.ScoreDefinitionTypeModel;
+import org.optaplanner.workbench.screens.solver.model.ScoreDirectorFactoryConfigModel;
+import org.optaplanner.workbench.screens.solver.model.SolverConfigModel;
+import org.optaplanner.workbench.screens.solver.model.TerminationConfigModel;
+
+import static org.junit.Assert.assertEquals;
+
+public class ToSolverConfigTest {
+
+    @Test
+    public void get() {
+        ToSolverConfig toSolverConfigModel = new ToSolverConfig( getSolverConfigModel() );
+        SolverConfig solverConfig = toSolverConfigModel.get();
+
+        assertEquals( Long.valueOf( 1 ), solverConfig.getTerminationConfig().getMillisecondsSpentLimit() );
```

Comment:
IMO, It'd be easier to read after assigning solverConfig.getTerminationConfig() to a local variable.